qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -851,9 +851,9 @@ def test_create_shorter_prep_template(self):
             self.metadata, self.test_study, self.data_type)
 
         # make sure the two samples were added correctly
-        self.assertEqual(pt.id, 2)
+        self.assertEqual(pt.id, 3)
```

Comment:
Can you remove this line? From previous discussions I think we agreed that we don't care about the specific ids